pkg/kudo/operator.go

```diff
@@ -205,15 +222,74 @@ func (operator Operator) Uninstall() error {
 		Delete(operator.Operator.Name, &options)
 	if err != nil {
 		return fmt.Errorf(
-			"failed to delete Operator %s in namespace %s: %w",
+			"failed to delete Operator %s in namespace %s: %v",
 			operator.Operator.Name,
 			operator.Operator.Namespace,
 			err)
 	}
 
+	if timeout != 0 {
+		kudoV1beta1 := operator.client.Kudo.KudoV1beta1()
```

Comment:
Nit: Let's name this `kudoClient`. IMO `kudoV1beta1` is easy to mistake with `kudov1beta1` which makes this harder to read.